packages/web-components/src/presence-badge/README.md

```diff
@@ -0,0 +1,116 @@
+---
+id: card
+title: fluent-card
+sidebar_label: card
+custom_edit_url: https://github.com/microsoft/fluentui/edit/master/packages/web-components/src/card/README.md
+---
+
+# fluent-presence-badge
+
+## Overview
+
+A `fluent-presence-badge` is designed to be used with the Avatar control to display a status icon based on the in/out/busy status of an Office user.
+
+## API
+
+Extends FAST Element
+
+_Component Name_
+
+- `fluent-presence-badge`
+
+_Attributes_
+
+- `status` - string value that sets the icon to display: `"available" | "away" | "busy" | "Do not disturb" | "offline" | "blocked"`
+- `outofoffice` - boolean value that shows user's in/out of office status and visually overrides the status icon for `"away"` and `"offline"` and switches other status icons to the fluent outline version
+- `title` - A localized string that concatenates a sentence case status and the out-of-office setting, for example, "Do not disturb, Out of office"
+
+**Events**
+
+- None
+
+## Anatomy and appearance
+
+```html
+<!-- shadow root -->
+<div aria-hidden="true" class="fluent-presence status-icon do-not-disturb">
+  <slot name="do-not-disturb" title="Do not disturb">
+    <svg><path></path></svg>
+  </slot>
+</div>
+<!-- end shadow root -->
+```
+
+_Slot names_
+
+- available
+- away
+- busy
+- do-not-disturb
+- offline
+- unknown
+- blocked
+- out-of-office
+
+_Host classes_
+
+- available
+- away
+- busy
+- do-not-disturb
+- offline
+- unknown
+- blocked
+- out-of-office
+
+## CSS overrides
+
+The badge can be customized several ways:
+
+- icon size and color can be changed
+  - `--presence-size: 16px` can be changed
+  - Color can be changed for each status:
+    - `--presence-color-available`
+    - `--presence-color-away`
+    - `--presence-color-busy`
+    - `--presence-color-do-no-disturb`
+    - `--presence-color-offline`
+    - `--presence-color-unknown`
+    - `--presence-color-blocked`
+    - `--presence-color-out-of-office`
+- icon mask width and color
+  - `--presence-mask-width: 8px;`
+  - `--presence-mask-color: #464775`
+- icon glyph (svg) can be changed
+
+## Usage
+
+```html live
+<fast-design-system-provider use-defaults>
+  <!-- common usage -->
+  <fluent-presence-badge status="busy" title="Busy"></fluent-presence-badge>
```

Comment:
Should the `title` default to the status unless it is set?

Reply:
There shouldn't be a title - the Fluent spec calls for the parent (typically Avatar) to host the title or alt text.

I'll change this in the readme.